.github/workflows/buildAndDeploy.yml

```diff
@@ -98,6 +104,12 @@ jobs:
         with:
           version: v3.10.0
 
+      - name: Add dependency chart repos
+        run: helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+
+      - name: Download chart dependencies before linting
```

Comment:
Unlike skaffold and the chart releaser tool, lint does not automatically download chart dependencies.